### PR TITLE
Add gulp mocha, a test task and get travis to run tests

### PR DESCRIPTION
**Pull request by gemmaleigh**
Add a basic sanity test suite to check that the app runs the build task (which copies assets to the correct directories and that the app runs).

Add a travis.yml file to get [Travis to run this test](https://travis-ci.org/alphagov/govuk_frontend_alpha_npm_test).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,8 +31,10 @@
     "serve-favicon": "^2.3.0"
   },
   "devDependencies": {
+    "assert": "^1.4.1",
     "connect-livereload": "^0.5.4",
     "del": "^2.2.0",
+    "fs": "0.0.1-security",
```
Comment (robinwhittleton) [inline package.json]: According to @nickcolley this is built in and doesn’t need to be added to the `package.json` manually.

**Comment by gemmaleigh** [comment]
@mcgoooo, want to review this one?

**Comment by mcgoooo** [comment]
@gemmaleigh on it

**Comment by dsingleton** [comment]
Testing this locally it works for me, and the Travis branch build passes.

Merging to get master passing at all - otherwise Travis tries to run the app in a ruby environment and fails.

**Pull request merged**